includes/class-linkbacks-handler.php

```diff
@@ -81,11 +81,6 @@ public static function register_meta() {
 	 * Update an Enhanced Comment
 	 */
 	public static function enhance( $commentdata, $comment = array(), $commentarr = array() ) {
-		if ( ! empty( $commentarr ) ) {
```

Comment:
Because the original reason for it is gone. The cutting of the array is now done lower in wp_update_comment as of 4.7.1